src/Statsd/Listener.php

```diff
@@ -33,17 +34,11 @@ public function __construct(Client $statsdClient, EventDispatcherInterface $even
      */
     public function onKernelException(ExceptionEvent $event)
     {
-        // @TODO: remove this backward compatibility layer after symfony 4.4 has been dropped
-        if (method_exists($event, 'getThrowable')) {
-            $exception = $event->getThrowable();
-        } else {
-            $exception = $event->getException();
-        }
+        $exception = $event->getThrowable();
 
+        $code = 'unknown';
```

Comment:
it does not look like a phpstan requirement to assign a variable twice instead of once

Reply:
@Oliboy50, do you prefer this ? 
```
$this->eventDispatcher->dispatch(
    new StatsdEvent($code ?? 'unknown'),
    'statsd.exception'
);
```

Reply:
yes I'd prefer, over both the new and the old implementations

because I think it's a bad idea to potentially assign 2 different types to a single variable

Reply:
I like more the way of implementation how it was proposed by @Benoit382 . Besides of that I think all notices had been resolved